packages/xstate-wallet/src/store/store.ts

```diff
@@ -390,6 +414,7 @@ export class XstateStore implements Store {
   public async getEntry(channelId: string): Promise<ChannelStoreEntry> {
     const entry = await this.backend.getChannel(channelId);
     if (!entry) {
+      // FIXME: (liam) should be an Error type, not string
       throw 'Channel id not found';
```

Comment:
```suggestion
      throw new Error('Channel id not found');
```

Reply:
Regardless, could we choose a consistent keyword: `TODO` or `FIXME`?

Reply:
`FIXME` usually just means, to me at least, "do this later before you merge but don't worry about now"